src/MPDeviceBleImpl.cpp

```diff
@@ -467,7 +467,7 @@ void MPDeviceBleImpl::getCredential(const QString& service, const QString& login
         jobs = new AsyncJobs(getCred, reqid, this);
     }
 
-    jobs->append(new MPCommandJob(mpDev, MPCmd::GET_CREDENTIAL, createGetCredMessage(service, login),
+    jobs->append(new MPCommandJob(mpDev, MPCmd::GET_TOTP_CODE, createGetCredMessage(service, login),
```

Comment:
i'm guessing this is not wanted? :D

Reply:
oh, nope  (this remained from testing :D)